src/main/scala/shell/PCIeOverlay.scala

```diff
@@ -23,6 +23,8 @@ case class PCIeOverlayOutput(
 trait PCIeShellPlacer[Shell] extends ShellPlacer[PCIeDesignInput, PCIeShellInput, PCIeOverlayOutput]
 
 case object PCIeOverlayKey extends Field[Seq[DesignPlacer[PCIeDesignInput, PCIeShellInput, PCIeOverlayOutput]]](Nil)
+case object PCIeEdgeOverlayKey extends Field[Seq[DesignPlacer[PCIeDesignInput, PCIeShellInput, PCIeOverlayOutput]]](Nil)
+case object PCIeFMCOverlayKey extends Field[Seq[DesignPlacer[PCIeDesignInput, PCIeShellInput, PCIeOverlayOutput]]](Nil)
```

Comment:
Why do these need to be separate if they are the same definition?

Reply:
I've reverted it to keep PCIeOverlayKey.